src/astro/sql/operators/drop.py

```diff
@@ -17,14 +19,15 @@ def __init__(
         **kwargs,
     ):
         self.table = table
-        task_id = task_id or get_unique_task_id(table.name + "_truncate")
+        task_id = task_id or get_unique_task_id("_drop")
         super().__init__(
             task_id=task_id,
             **kwargs,
         )
 
-    def execute(self, context: Dict) -> None:  # skipcq: PYL-W0613
+    def execute(self, context: Dict) -> Table:  # skipcq: PYL-W0613
         """Method run when the Airflow runner calls the operator."""
         database = create_database(self.table.conn_id)
         self.table = database.populate_table_metadata(self.table)
         database.drop_table(self.table)
+        return self.table
```

Comment:
Why do we need to return `Table` here?

Reply:
+1 table objects will mostly be useless since the table is deleted in DB. 

@dimberman is there any case where you think this will be useful?

Reply:
@tatiana maybe this would be a good use for the same type of operator we discussed for the raw_sql? Like an EmptyTable or something?

Reply:
@tatiana I changed it to a "DropTableResult". There needs to be SOMETHING returned for Xcom to work (@kaxil not sure if this a bug in Airflow we should consider).

Reply:
@dimberman I chatted to @ashb about this, since @kaxil is off this week.

I have a better understanding of this issue now. It feels, for now, the best option is indeed to return something - so we can extend the existing Task Flow API.